### ISSUE TITLE
[Support]: Illegal instruction during "npm run build" on Ubuntu Focal

### ISSUE THREAD

**Issue by FreemoX** [sinamics/ztnet#203]
### 📝 Describe the Problem You Are Having

Ran the `ztnet.sh` installation script as instructed by the docs on a semi-fresh Ubuntu Focal server, but the script fails during the "npm run build" part:

```
All migrations have been successfully applied.
Environment variables loaded from .env
Running seed command `ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts` ...
Seeding:: User Options complete!
Seeding:: Updating user ID complete!

🌱  The seed command has been executed.

> ztnet@0.1.0 build
> next build

- info Loaded env from /tmp/ztnet/.env
- warn You have enabled experimental feature (instrumentationHook) in next.config.mjs.
- warn Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

- info Skipping linting
- info Checking validity of types  
- info Creating an optimized production build ...Illegal instruction (core dumped)
```

### 🔖 Version

8ba6e6b

### 💻 Operating System

Ubuntu

### 📚 Any Other Information That May Be Helpful

The server runs an existing instance of Nginx Proxy Manager, not sure if relevant

**Comment by sinamics** [comment]
which version of node is installed? `node -v`

**Comment by sinamics** [comment]
Just tested `Linux ubuntu-focal 6.2.16-12-pve` and i run these installations instructions
https://ztnet.network/installation/linux
Everything went just fine. 

```bash
NAME="Ubuntu"
VERSION="20.04 LTS (Focal Fossa)"
```

Could you please inform what the exact steps you did so i can try to reproduce the issue.

**Comment by tinola** [comment]
I did the same test on VM environment, but with Ubuntu Server 20.04 LTS version (no X11) - I can confirm that everything works all right as a standalone installation. Just follow the instructions, especially make sure you have met the conditions in "1. Prerequisites"...

**Comment by FreemoX** [comment]
Node is v. 18.18.2, and is capable of building other packages. And yes, I've checked the prerequisites ... 

> Could you please inform what the exact steps you did so i can try to reproduce the issue.

I followed the instructions in the docs "https://ztnet.network/installation/linux", using a physical server only running Nginx Proxy Manager in a docker (otherwise fully fresh install). 
1) Both sudo and curl are installed and available.
2) Node and PostgreSQL are already installed under versions 18.18.2 and 12.16 respectively.
3) I then curled the install script and ran it with sudo from my users home directory.

No issues at any stage during the interactive installer until it attempts to build it. 

If no one else are having this issue, then the only thing I can conclude with are either an incompatibility with Nginx Proxy Manager, or an outdated PostgreSQL is somehow causing issues

**Comment by FreemoX** [comment]
Can confirm, it works fine on a fresh VM, but still fails during building on my physical machine, even after a system-wide update and a re-install of node (18.18.2)
I did a full clone of the system and ran it as a VM on another machine, it worked as expected again.
Installed Ubuntu Jammy on an identical machine and tried installing it stand-alone again; it failed during the build phase again.

I ran the docker setup on the original machine, and got it up and working, so I'm not totally sure what's causing these issues, but unless more diagnostic is wanted, this issue can be closed.
